src/main/java/net/masterthought/cucumber/reducers/ReducingMethod.java

```diff
@@ -13,6 +13,34 @@
      */
     MERGE_FEATURES_BY_ID,
 
+    /**
+     * Merge features and scenarios from different JSON files of different runs
+     * into a single report by features' and scenarios' ids.
+     *
+     * Merging rules:
+     * - Every new feature which is not in the result list is appended to the end.
+     *
+     * - When the results list already has a feature with such Id then we go down and apply the rules below to the scenarios:
+     *
+     *      1. if there is no scenario with a given Id in the feature's elements list
+     *         then add the scenario to the end of the list.
+     *
+     *      2. if scenario has a background (which is a previous element in the elements list)
+     *        then both elements are added to the end of the current feature's elements list.
```

Comment:
to the end or next to the background element?

Reply:
Doc has been updated.

in the case you asked the elements are added to the end of the list because the features' file has the following structure and it's possible to add a new data easily to the end:
- background
- scenario
- scenario
- background
- scenario
- new-background
- new-scenario